docs/faqs.md

```diff
@@ -319,11 +319,17 @@ features that will be broken unless you also upgrade Conan client (and sometimes
 need to modify your project if the recipe changes the binaries, flags,... it provides).
 
 To isolate from this changes there are different strategies you can follow:
+
+The minimum solution involves small changes to your Conan client configuration by
+
+ * **Pin the version of every reference you consume in your project** using recipe [revisions](https://docs.conan.io/en/latest/versioning/revisions.html)
+   and [lockfiles](https://docs.conan.io/en/latest/versioning/lockfiles/introduction.html).
```

Comment:
```suggestion
* **Pin the version of every reference you consume in your project** using either:
  * r[recipe revision (RREV)](https://docs.conan.io/en/latest/versioning/revisions.html): `foo/1.0@#RREV` instead of `foo/1.0`
  * [lockfiles](https://docs.conan.io/en/latest/versioning/lockfiles/introduction.html)
```

More explicit, new comers can easily get confused by RREV and PREV.